src/python/grpcio_tests/tests/unit/beta/_beta_features_test.py

```diff
@@ -65,7 +65,7 @@ def unary_stream(self, request, context):
             self._serviced = True
             self._condition.notify_all()
             return
-            yield
+            yield  # pylint: disable=unreachable
```

Comment:
@nathanielmanistaatgoogle As chief Pythonista here, is there a canonical way to write a generator function that is supposed to return an empty sequence than this ugly trick?

Reply:
This is the best way to do this and I just [haven't yet found the time to push back on the pylint folks](https://github.com/PyCQA/pylint/issues/1471). But they are wrong for complaining about this.